docs/sources/mimir/release-notes/v2.9.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Grafana Mimir version 2.9 release notes"
+menuTitle: "V2.9 release notes"
+description: "Release notes for Grafana Mimir version 2.9"
+weight: 300
+---
+
+# Grafana Mimir version 2.9 release notes
+
+Grafana Labs is excited to announce version 2.9 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Experimental support for optimized series selection strategy** Mimir can now use an optimized strategy for fetching posting lists that can significantly reduce memory allocations in the store-gateway.
+- **Reduced CPU utilisation for some shuffle sharding scenarios** Mimir queriers will now use significantly less CPU in cases where shuffle sharding is enabled with a high number of shards.
+- **Reduced object storage API calls in compactors and rulers** Mimir 2.9 comes with optimizations that will reduce the amount of times compactors and rulers need to access rules stored in object storage.
+- **Experimental support for streaming chunks from ingester to querier** This is expected to greatly reduce querier memory consumption when evaluating queries that select a large number of chunks, because chunks streamed from the querier can now be read into memory as needed.
```

Comment:
This will only be beneficial for queries that select many series (we still load all chunks for a single series into memory at once)

```suggestion
- **Experimental support for streaming chunks from ingester to querier** This is expected to greatly reduce querier memory consumption when evaluating queries that select a large number of series, because chunks streamed from the querier can now be read into memory as needed.
```